includes/Splitter.php

```diff
@@ -139,7 +139,7 @@ private function buildSplitter() {
 			'<div.*?>|<\/div>|' .                       // attributes of div elements
 			'<span.*?>|<\/span>|' .                     // attributes of span elements
 			'<file>[^<]*<\/file>|' .                    // stuff inside file elements
-			'style=".+?"|class=".+?"|' .                // styles and classes (e.g. of wikitables)
+			'style=".+?"|class=".+?"|data-sort-value=".+?"|' . 	// styles and classes (e.g. of wikitables)
```

Comment:
More generic variant could be:
`'style=".+?"|class=".+?"|data-[^=]+=".+?"|'`
?

Reply:
Hi, I don't have much time to test it, but what about something like this:
`<[^>]*>`
It would match all HTML Tags with or wihtout attributes.
Is there any case in which you have to replace anything inside a HTML tag?

Reply:
*push*